src/main/scala/tile/RocketTile.scala

```diff
@@ -111,13 +118,6 @@ class RocketTile private(
       omDTIM.orElse(omDCache)
     }
 
-    def getInterruptTargets(): Seq[OMInterruptTarget] = {
-      Seq(OMInterruptTarget(
-        hartId = rocketParams.hartId,
-        modes = OMModes.getModes(rocketParams.core.useVM)
-      ))
-    }
-
     def getOMRocketCores(resourceBindingsMap: ResourceBindingsMap): Seq[OMRocketCore] = {
```

Comment:
How come you were able to remove `getInterruptTargets` here but not `getOMRocketCores ` or `getOMICacheFromBindings`? Can't you just use the `RocketLogicalTreeNode` version like you did for the CLINT and Debug?

Reply:
Removed getOMRocketCores